src/Synergism.ts

```diff
@@ -1789,12 +1789,7 @@ export const updateAllMultiplier = (): void => {
     a *= (1 + player.achievements[58] / 100)
     a *= (1 + player.achievements[59] / 100)
     a *= Math.pow(1.01, player.upgrades[21] + player.upgrades[22] + player.upgrades[23] + player.upgrades[24] + player.upgrades[25])
-    if (player.upgrades[34] > 0.5) {
-        a *= 1.03 * 100 / 100
-    }
-    if (player.upgrades[35] > 0.5) {
-        a *= 1.05 / 1.03 * 100 / 100
-    }
+	a *= (1 + 1.03 * player.upgrades[34] + 1.02 * player.upgrades[35])
```

Comment:
This rewrite is not correct, as this would give a 3.05x multiplier when both upgrades are bought, but intended is 1.05x.